Dockerfile

```diff
@@ -33,6 +33,7 @@ RUN apt-get update && apt-get install -y \
 	automake \
 	bash-completion \
 	btrfs-tools \
+        bridge-utils \
```

Comment:
@LK4D4 All i need was brctl to delete an existing bridge. bridge-utils satisfies that need.